wp-content/themes/csisnuclearnetwork/assets/_scss/layout/_footer.scss

```diff
@@ -132,45 +167,67 @@
     }
   }
 
-  &__newsletter {
-    @include breakpoint('small') {
-      margin-right: rem(4);
+  .newsletter-block {
+    heading,
```

Comment:
We should specify what type of `heading`.  In this case it should be an `h3`.  There are a few other places where this should be updated as well.